docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -680,6 +680,25 @@ topic name, column names and associated types, serialization formats, queries th
 read and write from the stream or table, and more. For more details, see the
 [API reference](api/io/confluent/ksql/api/client/Client.html#describeSource(java.lang.String)).
 
+Connect to a {{ site.ccloud }} ksqlDB cluster <a name="connect-to-cloud"></a>
```

Comment:
Inserting this here separates the example describe source command below from the textual description. Did you mean to insert this after the example instead?

Reply:
D'oh! Moved to the end of the topic.